src/detect-pangram.js

```diff
@@ -15,7 +15,18 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
-}
+  if (phrase.length < 26) {
```

Comment:
From my perspective , you should use a variable for number 26

Reply:
create a variable for 26